src/databricks/labs/ucx/source_code/queries.py

```diff
@@ -1,15 +1,17 @@
 from collections.abc import Iterable
 
 import sqlglot
-from sqlglot.expressions import Table
-
-from databricks.labs.ucx.hive_metastore.table_migrate import MigrationIndex
+from sqlglot import ParseError
+from sqlglot.expressions import Table, Expression
+from databricks.labs.ucx.hive_metastore.migration_status import MigrationIndex
+import databricks.labs.ucx.hive_metastore.tables as ucx_tables
```

Comment:
```suggestion
import databricks.labs.ucx.hive_metastore as tables
```
otherwise it looks odd

Reply:
Done